src/test/java/gov/usgs/wma/waterdata/collections/FeaturesParamsIT.java

```diff
@@ -88,25 +88,25 @@ public void noNextLinkTest() {
 	@Test
 	public void paramBoundsTest() {
 		for (String collectionId : COLLECTION_IDS) {
-			String featuresJsonStr = doCollectionRequest("/collections/" + collectionId + "/items?limit=0",
+			String rtn = doCollectionRequest("/collections/" + collectionId + "/items?startIndex=-1",
 					HttpStatus.NOT_FOUND.value());
```

Comment:
Nice refactor helper method. Could we make it take the constant an it the method it calls the value() method?

Reply:
Good suggestion, improves readability and usability not to have to call HttpStatus.value() method when invoking doCollectionRequest()